### ISSUE TITLE
StepUpBar question

### ISSUE THREAD

**Issue by cwiese** [Imaginea/uvCharts#128]
Can the StepUpBar chart do this:

i.e. can we step up to a point and then start over from zero to start another "step up" ??

**Comment by hashd** [comment]
No, not at this point. But I can add this feature tomorrow, should be fairly straight forward

User can mark a reset along with the value when he wants to reset the sum to 0.

**Comment by cwiese** [comment]
Yes that sounds perfect !

**Comment by hashd** [comment]
There is a new release https://github.com/Imaginea/uvCharts/releases which introduces this feature. Do checkout and let us know if it works as expected.
